src/base/postcss.config.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
   plugins: [
     require("postcss-import"),
     require("tailwindcss/nesting"),
+    require("tailwindcss")("./src/base/tailwind.config.js"),
```

Comment:
file `./src/base/tailwind.config.js` doesn't exist.
Can you add it with following content?
```
module.exports = {
  theme: {
    colors: require("../theming"),
  },
}
```

Reply:
Good catch - my bad